### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 5)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by msewaweru** [comment]
Trigger rebuild.

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.md [hunk 1]
```diff
@@ -40,19 +40,19 @@ PS C:\> Get-AzureADMSRoleAssignment -Filter "roleDefinitionId eq '62e90394-69f5-
 
 RoleDefinitionId                     PrincipalId                          ResourceScope Id
 ----------------                     -----------                          ------------- --
-62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
+62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.md:43]: Not all PrincipalId values have been updated.

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.yml [hunk 2]
```diff
@@ -22,18 +22,18 @@ examples:
 
     RoleDefinitionId                     PrincipalId                          ResourceScope Id
     ----------------                     -----------                          ------------- --
-    62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
+    62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.yml:25]: Not all PrincipalId values have been updated.

**Review by msewaweru — APPROVED** [review]
Some GUIDs haven't been updated.

**Review by msewaweru — CHANGES_REQUESTED** [review]
Some GUIDs haven't been updated.

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.yml [hunk 3]
```diff
@@ -22,18 +22,18 @@ examples:
...
   code: |-
-    PS C:\> Get-AzureADMSRoleAssignment -Filter "principalId eq '69584002-b4d1-4055-9c94-320542efd653'"
+    PS C:\> Get-AzureADMSRoleAssignment -Filter "principalId eq 'aaaaaaaa-bbbb-cccc-1111-222222222222'"
 
     RoleDefinitionId                     PrincipalId                          ResourceScope Id
     ----------------                     -----------                          ------------- --
-    89c55b63-78c3-478b-b79e-074d0e87269e 69584002-b4d1-4055-9c94-320542efd653 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
-    62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
+    89c55b63-78c3-478b-b79e-074d0e87269e aaaaaaaa-bbbb-cccc-1111-222222222222 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
+    62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.yml:36]:
```suggestion
    89c55b63-78c3-478b-b79e-074d0e87269e 69584002-b4d1-4055-9c94-320542efd653 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
    62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.yml (hunk 2 above)
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.yml:25]:
```suggestion
    62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRqAVAnJQyBBLv1lM-1
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.md [hunk 4]
```diff
@@ -40,19 +40,19 @@ PS C:\> Get-AzureADMSRoleAssignment -Filter "roleDefinitionId eq '62e90394-69f5-
...
+PS C:\> Get-AzureADMSRoleAssignment -Filter "principalId eq 'aaaaaaaa-bbbb-cccc-1111-222222222222'"
 
 RoleDefinitionId                     PrincipalId                          ResourceScope Id
 ----------------                     -----------                          ------------- --
-89c55b63-78c3-478b-b79e-074d0e87269e 69584002-b4d1-4055-9c94-320542efd653 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
-62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
-eb1d8c90-acf5-460d-8424-c1f1a6fbdb85 69584002-b4d1-4055-9c94-320542efd653 /             NIwd6_WsDUaEJMHxpvvbhQJAWGnRtGDAnJQyBBLv1lM-1
+89c55b63-78c3-478b-b79e-074d0e87269e aaaaaaaa-bbbb-cccc-1111-222222222222 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
+62e90394-69f5-4237-9190-012177145e34 aaaaaaaa-bbbb-cccc-1111-222222222222 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
+eb1d8c90-acf5-460d-8424-c1f1a6fbdb85 aaaaaaaa-bbbb-cccc-1111-222222222222 /             NIwd6_WsDUaEJMHxpvvbhQJAWGnRtGDAnJQyBBLv1lM-1
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSRoleAssignment.md:55]: This shows up in the GUID list to be remediated, but will revert the table
```suggestion
89c55b63-78c3-478b-b79e-074d0e87269e 69584002-b4d1-4055-9c94-320542efd653 /             Y1vFBcN4i0e3ngdNDocmngJAWGnRtGDAnJQyBBLv1lM-1
62e90394-69f5-4237-9190-012177145e34 69584002-b4d1-4055-9c94-320542efd653 /             lAPpYvVpN0KRkAEhdxReEAJAWGnRtGDAnJQyBBLv1lM-1
eb1d8c90-acf5-460d-8424-c1f1a6fbdb85 69584002-b4d1-4055-9c94-320542efd653 /             NIwd6_WsDUaEJMHxpvvbhQJAWGnRtGDAnJQyBBLv1lM-1

```

**Review by cilwerner — CHANGES_REQUESTED** [review]
@msewaweru, please commit and merge

**Review by msewaweru — APPROVED** [review]

**Pull request merged**